lib/srv/regular/sftp.go

```diff
@@ -127,9 +130,22 @@ func (s *sftpSubsys) Start(ctx context.Context,
 	if err != nil {
 		return trace.Wrap(err)
 	}
-	// TODO: put in cgroup?
 	execRequest.Continue()
 
+	// Send the file transfer request if applicable
+	encodedReq := []byte{0x0}
```

Comment:
Can you explain why we initialize this to a 1-byte array?

Reply:
Added comment that explains this. The file transfer request ends in a null byte so if we don't have any request to send just sending a null byte lets the SFTP process no that there's no file transfer request.